pkg/reconcilermanager/controllers/reposync_controller.go

```diff
@@ -944,6 +944,7 @@ func (r *RepoSyncReconciler) upsertRoleBinding(ctx context.Context, reconcilerRe
 	childRB.Namespace = rbRef.Namespace
 
 	op, err := CreateOrUpdate(ctx, r.client, childRB, func() error {
+		core.AddLabels(childRB, labelMap)
```

Comment:
The same binding is used for many RepoSyncs, so it doesn't fit to apply the typical sync-kind/sync-name/sync-namespace label.

Reply:
deleted the sync name label